modules/sqs-queue/CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+## Upgrading to `v1.TODO.0`
```

Comment:
I believe we decided to just call this the PR # since we cant determine the version until it's released of course